lib/parsers/mp3_parser/id3_extraction.rb

```diff
@@ -1,6 +1,9 @@
 module FormatParser::MP3Parser::ID3Extraction
   ID3V1_TAG_SIZE_BYTES = 128
-  ID3V2_TAG_VERSIONS = ["\x43\x00".b, "\x03\x00".b, "\x02\x00".b]
+  # it supports 2.4.x, 2.3.x, 2.2.x which are supported by the gem id3tag
+  # see https://id3.org/Developer%20Information for more details of each version
+  # I'm not sure why it has the version 2.67 listed, but it broke 2 tests when I tried to remove it
```

Comment:
Do we need to leave this comment here?

Reply:
@julik could you help with that pls? do you know the reason of this number 67?

Reply:
FWIW I could not find any online sources referencing this mythical 2.67 version (also searched for the hexadecimal notation).

It was introduced in format_parser with #115 as a supported version and in an (seemingly, to me) unrelated test without further info on this version number.

So if @julik can't remember, I would suggest removing it (and changing the affected test to some other supported version)? In a separate PR though.

I agree with @linkyndy that this comment should be removed from this PR:
```ruby
# I'm not sure why it has the version 2.67 listed, but it broke 2 tests when I tried to remove it
```

Reply:
I opened https://github.com/WeTransfer/format_parser/pull/170 to remove it and rebased this branch with this other, ok?